flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/model/node/ActionCall.java

```diff
@@ -59,9 +59,9 @@ public Map<Path, List<TextEdit>> toSource(SourceBuilder sourceBuilder) {
             sourceBuilder.token().keyword(SyntaxKind.RETURN_KEYWORD);
         }
 
-        if (sourceBuilder.flowNode.hasFlag(FlowNode.NODE_FLAG_CHECKED)) {
+//        if (sourceBuilder.flowNode.hasFlag(FlowNode.NODE_FLAG_CHECKED)) {
             sourceBuilder.token().keyword(SyntaxKind.CHECK_KEYWORD);
-        }
+//        }
```

Comment:
Do we need to keep this commented code?

Reply:
This is a hack we are doing right now. Once the index is available we should get the correct flag and use the flag to add the check.

Reply:
Ack. Even in that case we better add a TODO instead of commenting out the code blocks IMO